src/core/server/saved_objects/export/saved_objects_exporter.ts

```diff
@@ -6,9 +6,9 @@
  * Side Public License, v 1.
  */
 
+import { Readable } from 'stream';
```

Comment:
nit: `import type` because `Readable` is also a class and we may risk importing the runtime code?